### PR TITLE
speedup CI

### PR DESCRIPTION
**Pull request by matklad**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +16,7 @@ matrix:
           script:
               - rustup component add rustfmt
               - rustup component add rust-src
+              - printf "\n\n[profile.dev]\ndebug = false\nopt-level = 1\n" >> Cargo.toml
```
Comment (lnicola) [inline .travis.yml]: :cry:

Comment (matklad) [reply .travis.yml]: Putting bash into YAML is definitely not my favorite thing :D

Comment (lnicola) [reply .travis.yml]: I'm still running tests for #1561 :laughing:.

**Comment by matklad** [comment]
Ok, numbers in #1561 look super sweet, let's just merge this then!

bors r+

Thanks @lnicola !

**Pull request merged**